src/client/service/nodesAdd.ts

```diff
@@ -79,7 +79,6 @@ function nodesAdd({
             host,
             port,
           } as NodeAddress,
-          true,
```

Comment:
What is the impact of this change?

Reply:
Blocking has been added back in. This change has been reverted as a result.